src/drr/DRR.h

```diff
@@ -0,0 +1,335 @@
+/* +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    Copyright (c) 2017 of Haochuan Chen
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ */
+// Build requirement: boost, c++11 compatible compiler.
+#ifdef __PLUMED_HAS_BOOST_SERIALIZATION
```

Comment:
@HanatoK I think plumedcheck wants the guard to be the first preprocessor command. Anyway, you can generate guards using a script (more later in the general comments)